transcom/mymove
# B 17588 enable prime to add partial weight

Ticket: B-17588

This PR will allow the prime to enter in a billable weight for each serviceitem. The weight will be entered as a key value pair with key: WeightBilled and value as the weight.

This change is for the following serviceItems:
DLH
DSH
DUPK
DNPK
DOFSIT
DOASIT 
DOPSIT
DDDSIT 
DDSHUT 
DOSHUT 
FSC 
DDASIT 

To test:
Submit a payment request with at least one of the service items listed above. Make sure the service Item has the params populated with a WeightBilled key value pair, where key is equals WeightBilled and value is the requested weight for the service Item. Submit the payment request.

Go to the TIO View and view the payment request.
View the serviceItems and notice the cwt weight. 

I do not mind huddling up and walking through how to test this. It is tricky.
